owl/SOMA-PYCRAM.owl

```diff
@@ -0,0 +1,66 @@
+- For object lookups: a dul:Configuration of feature‐roles (e.g., type, color, shape).
+- For actions and motions: a dul:Method that prescribes how to carry out an underspecified task or movement toward a goal.
+
+An underspecified PyCRAM designator will typically be resolved during a PyCRAM Resolution Task; the result is a new PyCRAM designator with more concrete descriptions; possibly even a reference to a single physical object.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator> "PyCRAM Designator")
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator> <http://www.ease-crc.org/ont/SOMA.owl#Structured_Text>)
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator> ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#expresses> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Description>))
+
+# Class: <http://www.ease-crc.org/ont/SOMA.owl#PyCramMotionDesignator> (PyCRAM Motion Designator)
+
+AnnotationAssertion(rdfs:comment <http://www.ease-crc.org/ont/SOMA.owl#PyCramMotionDesignator> "A PyCRAM Designator expressing a Motion Description.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA.owl#PyCramMotionDesignator> "PyCRAM Motion Designator")
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramMotionDesignator> <http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator>)
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramMotionDesignator> ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#expresses> <http://www.ease-crc.org/ont/SOMA.owl#MotionDescription>))
+
+# Class: <http://www.ease-crc.org/ont/SOMA.owl#PyCramObjectDesignator> (PyCRAM Object Designator)
+
+AnnotationAssertion(rdfs:comment <http://www.ease-crc.org/ont/SOMA.owl#PyCramObjectDesignator> "A PyCRAM Designator expressing the Configuration of an Object.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA.owl#PyCramObjectDesignator> "PyCRAM Object Designator")
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramObjectDesignator> <http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator>)
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#PyCramObjectDesignator> ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#expresses> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Configuration>))
+
+# Class: <http://www.ease-crc.org/ont/SOMA.owl#Resolving_of_PyCRAM_Designators> (Resolving a PyCRAM Designator)
+
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA.owl#Resolving_of_PyCRAM_Designators> "Resolving a PyCRAM Designator")
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#Resolving_of_PyCRAM_Designators> <http://www.ease-crc.org/ont/SOMA.owl#DerivingInformation>)
+SubClassOf(<http://www.ease-crc.org/ont/SOMA.owl#Resolving_of_PyCRAM_Designators> ObjectSomeValuesFrom(<http://www.ease-crc.org/ont/SOMA.owl#isTaskOfInputRole> <http://www.ease-crc.org/ont/SOMA.owl#PyCramDesignator>))
```

Comment:
What are the domains and ranges of the isTaskOfInput/OutputRole?

Reply:
Ah damn, I forgot the Role inbetween. Will change that, good catch!